test/jdk/java/lang/ProcessBuilder/JspawnhelperWarnings.java

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright (c) 2024, Oracle and/or its affiliates. All rights reserved.
+ * DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS FILE HEADER.
+ *
+ * This code is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License version 2 only, as
+ * published by the Free Software Foundation.  Oracle designates this
+ * particular file as subject to the "Classpath" exception as provided
+ * by Oracle in the LICENSE file that accompanied this code.
+ *
+ * This code is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+ * version 2 for more details (a copy is included in the LICENSE file that
+ * accompanied this code).
+ *
+ * You should have received a copy of the GNU General Public License version
+ * 2 along with this work; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA.
+ *
+ * Please contact Oracle, 500 Oracle Parkway, Redwood Shores, CA 94065 USA
+ * or visit www.oracle.com if you need additional information or have any
+ * questions.
+ */
+
+/*
+ * @test
+ * @bug 8325567
+ * @requires (os.family == "linux") | (os.family == "aix")
+ * @library /test/lib
+ * @run main/othervm/timeout=300 JspawnhelperWarnings
+ */
+
+import java.nio.file.Paths;
+import java.util.Arrays;
+import java.util.concurrent.TimeUnit;
+
+import jdk.test.lib.process.OutputAnalyzer;
+import jdk.test.lib.process.ProcessTools;
+
+public class JspawnhelperWarnings {
+
+    private static final int TIMEOUT = 60;
+
+    private static void jspawnhelperWithNArgs(int nArgs) throws Exception {
+        System.out.println("Running jspawnhelper with "+nArgs+" args");
+        String[] args = new String[nArgs +1];
+        Arrays.fill(args, "1");
+        args[0] = Paths.get(System.getProperty("java.home"), "lib", "jspawnhelper").toString();
+        Process p = ProcessTools.startProcess("jspawnhelper", new ProcessBuilder(args));
+        OutputAnalyzer oa = new OutputAnalyzer(p);
+        if (!p.waitFor(TIMEOUT, TimeUnit.SECONDS)) {
+            throw new RuntimeException("jspawnhelper timed out after " + TIMEOUT + " seconds");
+        }
+        oa.shouldHaveExitValue(1);
+        oa.shouldContain("This command is not for general use");
+    }
+
+     public static void main(String[] args) throws Exception {
+        for (int nArgs = 0; nArgs < 10; ++nArgs)
+            jspawnhelperWithNArgs(nArgs);
```

Comment:
Style: braces for loop body. There is also no point in using `++nArgs`, when `nArgs++` is sufficient.